.github/workflows/CI.yml

```diff
@@ -189,6 +189,26 @@ jobs:
         run: ${{ matrix.extra }} bash dev/ci-prepare.sh
       - name: "Run tests"
         run: ${{ matrix.extra }} bash dev/ci.sh
+      - name: "Upload pdf manuals"
+        if: ${{ matrix.test-suites == 'makemanuals' }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: manuals-pdf
+          path: |
+            doc/dev/manual.pdf
+            doc/hpc/manual.pdf
+            doc/ref/manual.pdf
+            doc/tut/manual.pdf
+      - name: "Upload html manuals"
+        if: ${{ matrix.test-suites == 'makemanuals' }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: manuals-html
+          path: |
+            doc/dev/*.{html,css,js}
+            doc/hpc/*.{html,css,js}
+            doc/ref/*.{html,css,js}
```

Comment:
This didn't work - I think bash-style `{...}` patterns are not supported. But OTOH you could maybe use four patterns of the form `doc/*/*.html`

Reply:
That kind of thing seems to work fine, see e.g. the result https://github.com/wilfwilson/gap/runs/2057072844?check_suite_focus=true of commit https://github.com/wilfwilson/gap/commit/113226445d62c82f5805d33a69594fb7c1fc9474

The log gives:
```
With the provided path, there will be 290 files uploaded
Total file count: 290 ---- Processed file #276 (95.1%)
Total size of all the files uploaded is 3459174 bytes
Finished uploading artifact manuals-html. Reported size is 3459174 bytes. There were 0 items that failed to upload
Artifact manuals-html has been successfully uploaded!
```
You'd think that would make the archive be about 3.5 MB, but it's actually 17.8 MB for some reason. Still acceptable though, I think?

Reply:
Just updated my PR. IMO the 3.5MB vs 17.8MB is not a problem. We only do that once per run of the CI workflow.